08-collab.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Collaborating
+minutes: 25
+---
+> ## Learning Objectives {.objectives}
+>
+> *   Explain how to collaborate pushing to a common repo
+
+For the next step, get into pairs.
+Pick one of your repositories on GitHub to use for collaboration.
+
+> ## Practicing by yourself {.callout}
+>
+> If you're working through this lesson on your own, you can carry on by opening
+> a second terminal window, and switching to another directory (e.g. `/tmp`).
+> This window will represent your partner, working on another computer. You
+> won't need to give anyone access on GitHub, because both 'partners' are you.
```

Comment:
suggest "both partners are you, using your Github login" to make it clearer what is meant here.